widgets/mpd_all.lua

```diff
@@ -83,8 +117,8 @@ function mpd_all.async(format, warg, callback)
         ["{bitrate}"]  = 0,
         ["{elapsed}"]  = 0,
         ["{duration}"] = 0,
-        ["{repeat}"]   = false,
-        ["{random}"]   = false,
+        ["{repeat}"]   = 0,
+        ["{random}"]   = 0,
```

Comment:
This is a simple change to make sure data types stay the same, since mpd will send `0` or `1`, not `true` or `false`